ui/app/models/ldap/config.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Model, { attr } from '@ember-data/model';
+import { withFormFields } from 'vault/decorators/model-form-fields';
+import { withModelValidations } from 'vault/decorators/model-validations';
+
+const validations = {
+  binddn: [{ type: 'presence', message: 'Administrator Distinguished Name is required.' }],
+  bindpass: [{ type: 'presence', message: 'Administrator Password is required.' }],
+};
+const formGroups = [
+  { default: ['binddn', 'bindpass', 'url', 'password_policy'] },
+  { 'TLS options': ['starttls', 'insecure_tls', 'certificate', 'client_tls_cert', 'client_tls_key'] },
+  { 'More options': ['userdn', 'userattr', 'upndomain', 'connection_timeout', 'request_timeout'] },
+];
+
+@withModelValidations(validations)
+@withFormFields(null, formGroups)
+export default class KubernetesConfigModel extends Model {
+  @attr('string') backend; // dynamic path of secret -- set on response from value passed to queryRecord
+
+  @attr('string', {
+    label: 'Administrator Distinguished Name',
+    subText:
+      'Distinguished name of the adminstrator to bind (Bind DN) when performing user and group search. Example: cn=vault,ou=Users,dc=example,dc=com.',
+  })
+  binddn;
+
+  @attr('string', {
+    label: 'Administrator Password',
```

Comment:
Yep, I'm going to push for the validation messages to have only the first word capitalized given the label follows this pattern.

Reply:
This has both words capitalized though which is how it is in the designs. All good though I will update the validation messages.

Reply:
I think I just need to clarify for my own sanity with design. Lower down I see Connection timeout, lower case second word.

Reply:
A very long time ago I remember a document (RFC?) that outlined how and when we use capitalization in Vault. It seems to be lost to the ethers now, but the general TL;DR is use sentence case, and capitalize Vault specific things (like Vault, Vault Agent, etc). I'll try to track down that document because that would be a lot better than just my dim memory